### PR TITLE
Update shepherding roles and documentation

### PR DESCRIPTION
**Pull request by sedders123**
This will close https://github.com/srobo/tasks/issues/114

### CODE REVIEW

File: docs/competition/matches/shepherding.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ original:
 ---
 # Shepherding
 
-Anything related to shepherding goes here.
+The purpose of the shepherding team is to ensure that teams have the best chance possible of attending all of their matches, by getting to through staging before the start of their match.
```
Comment (Tyler-Ward) [inline docs/competition/matches/shepherding.md]: minor wording error "getting to through"

File: docs/competition/matches/shepherding.md [hunk 2]
```diff
@@ -19,41 +19,54 @@ Here is a description of the roles related to shepherding.
...
+
+### Roving Shepherd
+The responsibility of the roving shepherd is to sweep the venue and look for teams that are not in their pits.
+
+Teams are often known to set up 'bases' away from their pits, and the roving shepherd will be responsible for curating a list of these bases, to aid in the searching for teams that are not in their pits.
+
+The roving shepherd will be notified of teams that need to be found via radio, they will then check if they have a known base. If they do they'll proceed to it to try and find the team. If the team does not have a known base or is not there, the roving shepherd will walk through each floor of the arena in an attempt to locate them.
```
Comment (Tyler-Ward) [inline docs/competition/matches/shepherding.md]: arena isn't the right word here

File: docs/competition/matches/shepherding.md [hunk 3]
```diff
@@ -19,41 +19,54 @@ Here is a description of the roles related to shepherding.
...
+More specifically, the badgers should make sure that teams follow the direction of travel through the staging area. Once the teams have entered staging the badgers then hand out the robot badges and match keys to the teams, making sure that a team has staged in the correct slot and is receiving the correct badges. The teams leaving the arena should be advised to place their robots on the destaging desks for badge and USB key removal. The badgers need to collect the badges and keys and sort them, ready to be handed out in the next round of staging. The badgers need to ensure that teams do not leave the staging area with markers or USB-keys. The badgers should also clear any items left on the staging tables at this point to prepare for the next round of staging.
+
+### Roving Shepherd
```
Comment (PeterJCLaw) [inline docs/competition/matches/shepherding.md]: nit: markdown encourages a newline after headings.

(We don't have a style guide yet, but I'd like to introduce one and ideally a formatter too)

File: docs/competition/matches/shepherding.md [hunk 4]
```diff
@@ -19,41 +19,54 @@ Here is a description of the roles related to shepherding.
...
+
+Teams are often known to set up 'bases' away from their pits, and the roving shepherd will be responsible for curating a list of these bases, to aid in the searching for teams that are not in their pits.
+
+The roving shepherd will be notified of teams that need to be found via radio, they will then check if they have a known base. If they do they'll proceed to it to try and find the team. If the team does not have a known base or is not there, the roving shepherd will walk through each floor of the venue in an attempt to locate them.
+
+The Roving Shepherd is **not** required to find every missing team, purely do a best effort sweep of the venue. If a team can not be found they will notify the Head Shepherd immediately.
+
+## Communication
+Shepherd's will communicative via radio with a fallback option of Slack if for some reason radios stop working. While Radios will require brief volunteer training, it means that our primary communication method is not reliant on the venues wifi, volunteers having devices to view slack on, or volunteers using their own personal mobile data.
```
Comment (PeterJCLaw) [inline docs/competition/matches/shepherding.md]: suggestion: "While Radios" -> "While radios"

Comment (PeterJCLaw) [reply docs/competition/matches/shepherding.md]: also "venues wifi" -> "venue's wifi"

**Pull request merged**